app/views/participants/validations/what_is_your_trn.html.erb

```diff
@@ -0,0 +1,29 @@
+<% title = "What’s your teacher reference number (TRN)?" %>
+<% content_for :title, title %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl"><%= title %></h1>
+    <p class="govuk-body">This unique ID is:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>usually 7 digits long, for example ’4567814’</li>
```

Comment:
Are these smart quotes correct?

Reply:
No they don't look right, great spot!  I've let Brian know as I lifted that copy straight from the prototype